core/test/variable_operations_test.cpp

```diff
@@ -1279,6 +1280,45 @@ TEST(VariableTest,
   EXPECT_EQ(a, expected);
 }
 
+TEST(ATan2Test, atan2) {
+  auto x = makeVariable<double>(units::Unit(units::m), Values{1.0});
+  auto y = x;
+  auto expected = makeVariable<double>(units::Unit(units::rad),
+                                       Values{scipp::pi<double> / 4});
+  EXPECT_EQ(atan2(y, x), expected);
+}
+
+TEST(ATan2Test, no_variance_on_args) {
+  auto with_no_var = makeVariable<double>(units::Unit(units::m), Values{1.0});
+  auto with_var =
+      makeVariable<double>(units::Unit(units::m), Values{1.0}, Variances{1.0});
+  Variable out;
+  EXPECT_THROW((out = atan2(with_var, with_no_var)), except::VariancesError);
+  EXPECT_THROW((out = atan2(with_no_var, with_var)), except::VariancesError);
+}
```

Comment:
Should this rather be a test for the correct flag in the operator (in the element-test file), plus a test in `transform_test.cpp` ensuring that this exception is thrown if the flag is provided?